src/bridgestan.cpp

```diff
@@ -9,15 +9,14 @@ int bs_patch_version = BRIDGESTAN_PATCH;
 
 #include <sstream>
 
-bs_model_rng* bs_construct(const char* data, unsigned int seed,
-                           unsigned int chain_id, char** error_msg) {
+bs_model* bs_construct(const char* data, unsigned int seed, char** error_msg) {
```

Comment:
This PR proposes `bs_construct_rng` and `bs_destruct_rng`.

Is it worth better aligning the model and rng construct and destruct function names with the following changes?

`bs_construct` -> `bs_model_construct`
`bs_destruct` -> `bs_model_destruct` 
`bs_construct_rng` -> `bs_rng_construct`
`bs_destruct_rng` -> `bs_rng_destruct`

Thoughts?

Reply:
I agree